actions/create-and-push-image-manifests/action.yaml

```diff
@@ -0,0 +1,53 @@
+---
+description: Pulls image digests from GitHub and pushes manifests to GAR
```

Comment:
I don't think it would be too hard to make this not specific to Artifact Registry.

Mainly removing the login step up to a higher level.